### PR TITLE
Added missing Tools pages

### PR DESCRIPTION
**Pull request by pentabulge**
I've added the missing pages for the items in the "Tools" category

### CODE REVIEW

File: pages/Hercules'-Pickaxe.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hercules' Pickaxe is an item in the [tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tools) category.
+
+This tool is similar in function to an [Ore Crusher](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Crusher) or an [Electric Ore Grinder](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ore-Grinder): it crushes iron ore mined with it into [iron dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Iron-Dust), and gold ore into [gold dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Dust).
```
Comment (TheBusyBiscuit) [inline pages/Hercules'-Pickaxe.md]: You could add a header here.

And doesn't it also double other ore drops?
Or am I mxing things up here?

File: pages/Hercules'-Pickaxe.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Hercules' Pickaxe is an item in the [tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tools) category.
```
Comment (TheBusyBiscuit) [inline pages/Hercules'-Pickaxe.md]: Maybe add a "The" at the beginning.

File: pages/Explosive-Shovel.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+The Explosive Shovel is an item in the [tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tools) category.<br>
+It is an unenchanted diamond shovel which digs blocks in a 3x3x3 area.
+
+There is also a pickaxe equivalent of this tool: the [Explosive Pickaxe](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Explosive-Pickaxe).
```
Comment (TheBusyBiscuit) [inline pages/Explosive-Shovel.md]: You should probably add a header here.

File: pages/Explosive-Pickaxe.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+The Explosive Pickaxe is an item in the [tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tools) category.
+
+It is an unenchanted diamond pickaxe which mines blocks in a 3x3x3 area.<br>
+It is compatible with the Fortune enchantment, to get even more drops from your usual ore.
+
+There is also a shovel equivalent of this tool: the [Explosive Shovel](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Explosive-Shovel).
```
Comment (TheBusyBiscuit) [inline pages/Explosive-Pickaxe.md]: You should probably add a header here.

**Comment by pentabulge** [comment]
> You could add a header here.

And doesn't it also double other ore drops?
Or am I mxing things up here? 

It absolutely does, I just forgot to mention it.

File: pages/Hercules'-Pickaxe.md (hunk 1 above)
Comment (pentabulge) [reply pages/Hercules'-Pickaxe.md]: It absolutely does, I just forgot to mention it

**Pull request merged**